busybox.yaml

```diff
@@ -15,8 +15,12 @@ package:
         - /usr/bin
         - /usr/sbin
       script: |
-        #!/bin/busybox sh
-        /bin/busybox --install -s
+        #!/usr/bin/busybox sh
+        for i in `/usr/bin/busybox --list`; do
+          [ -e /usr/bin/$i ] && continue
+          [ -h /usr/bin/$i ] && continue
+          /usr/bin/busybox ln -s /bin/busybox /usr/bin/$i
+        done
```

Comment:
@kaniini so with this, apk --initdb and apko both create identical images for busybox only. Without needing to patch busybox.

still need to make this work in concert with coreutils.